packages/common/test/surveys/get-input-feature-service-model.test.ts

```diff
@@ -1,7 +1,7 @@
 /* Copyright (c) 2018 Environmental Systems Research Institute, Inc.
  * Apache-2.0 */
 
-import { getInputFeatureServiceModel } from "../../src/surveys/get-input-feature-service-model";
+import { getInputFeatureServiceModel } from "../../src/feedback/utils/get-input-feature-service-model";
```

Comment:
Thanks for consolidating the `surveys` and `feedback` dirs, though we may also want to move these tests from `test/surveys` => `test/feedback/utils` to better conform to our organizational pattern.